taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+        </li>
+      </ul>
+
+      <p>
+        As an Portal user, you should already have access to a TACC account through the
+        <a target="_blank" href="https://portal.tacc.utexas.edu/">TACC Portal</a>
+        . If you are unsure of your TACC account username, please consult the
+        “TACC Account Request Confirmation” email you have received from TACC Accounting Staff, or a relevant contact <samp>@tacc.utexas.edu</samp>.
+      </p>
+      <p>
+        TACC now requires MFA pairing. To set up MFA, you can follow
+        <a target="_blank" href="https://portal.tacc.utexas.edu/tutorials/multifactor-authentication">this tutorial</a>.
+      </p>
+      <h4 id="command_line_tools">
+        Using Command Line Tools to Transfer and Organize Data
+      </h4>
+
+      <p>
+        A common method of transferring files between TACC resources and/or your local machine is through the command line.
+      </p>
+      <h4 id="scp_sftp_rsync">
+        <code>scp</code>, <code>sftp</code>, &amp; <code>rsync</code>
+      </h4>
+
+      <p>
+        These three command line tools are secure and can be used to accomplish data transfer. You can run these commands directly from the terminal if your local system runs Linux or macOS.
+      </p>
+      <blockquote>
+        <p>
+          Note: It is possible to use these command line tools if your local machine runs Windows, but you will need to use a ssh client (ex.
+          <a target="_blank" href="https://www.putty.org/">PuTTY</a>
+          ).
+        </p>
+        <p>
+          However, for the scope of this project and ease in data transfer, it is recommended that Windows users follow our
+          <a href="#cyberduck_transfer">Transfer Data with Cyberduck</a>
+          guide below.
+        </p>
+      </blockquote>
+      <p>
+        For users that are new to the command line, using either <code>scp</code> or <code>sftp</code> to transfer data is advised.
+      </p>
+      <h4 id="data_transfer_prereqs">
+        Prerequisites for Data Transfer with Command Line Tools
+      </h4>
+
+      <p>Before we begin, you will need to know:</p>
+
+      <ul>
+        <li>the path to your data file(s) on your local system</li>
+        <li>the path to your transfer directory on the remote storage server</li>
+      </ul>
+
+      <h4 id="local_data_path">
+        Determining the Path to Your Data File(s) on Your Local System
+      </h4>
+
+      <p>
+        In order to transfer your project data, you will need to know where your file(s) is located on your local system. To do so, navigate through the file system on your local computer.
+      </p>
+      <p>
+        For example, you can do so on Mac by going to Finder. Common data file locations such as Desktop and Documents should be accessible through the Favorites pane on the left-hand side of the window. You can also do Finder &gt; Go to find other common locations on your computer.
+      </p>
+      <p>
```

Comment:
Once you have identified the location of the files, you can right-click on them and select either Get Info (on Mac) or Properties (on Windows) to view the path location on your local system.